tests/pageobjects/LobbyScreen.ts

```diff
@@ -0,0 +1,30 @@
+import PreMeetingScreen from './PreMeetingScreen';
+
+const DISPLAY_NAME_TEST_ID = 'lobby.nameField';
+const JOIN_BUTTON_TEST_ID = 'lobby.knockButton';
+
+/**
+ * Page object for the Lobby screen.
+ */
+export default class LobbyScreen extends PreMeetingScreen {
+    /**
+     * Returns the join button element.
+     */
+    getJoinButton(): ChainablePromiseElement {
```

Comment:
Is this method used somewhere? I don't seem to find any references.

Reply:
several places in the lobby test